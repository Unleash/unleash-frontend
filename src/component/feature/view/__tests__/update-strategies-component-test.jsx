import React from "react";
import UpdateStrategiesComponent from "../update-strategies-component";
import { shallow } from "enzyme/build";

it("render the create feature page", () => {
    let strategies = [{ name: "default" }];
    const tree = shallow(
        <UpdateStrategiesComponent
            featureToggleName="some-toggle"
            configuredStrategies={strategies}
            onSubmit={jest.fn()}
            setValue={jest.fn()}
            addStrategy={jest.fn()}
            removeStrategy={jest.fn()}
            moveStrategy={jest.fn()}
            onCancel={jest.fn()}
            updateStrategy={jest.fn()}
            validateName={jest.fn()}
            initCallRequired={false}
            title="title"
        />
    );
    expect(tree).toMatchSnapshot();
});
