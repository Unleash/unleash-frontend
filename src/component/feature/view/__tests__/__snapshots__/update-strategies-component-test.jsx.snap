// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the create feature page 1`] = `""`;
